Ansible/ui.yml

```diff
@@ -28,6 +28,8 @@
         - needs_certificate
         - emi_service
         - yaim_options
+        - site_name
+        - ronald
```

Comment:
I just tried to debug to be sure ansible was calling the right group_vars file. Fixing that